spec/features/tasks/create_task_spec.rb

```diff
@@ -56,7 +56,7 @@ def when_i_create_a_task_assigned_to_myself
     page.execute_script %Q{ $('#follow_up_datepicker').trigger("focus") }
     page.execute_script %Q{ $("td.day:contains('15')").trigger("click") }
     fill_in :task_body, with: "Test body"
-    click_button 'Save'
+    find("#new_task .modal-footer .btn-primary").click
```

Comment:
These don't necessarily need to be changed but I believe you can use `click_link` for a `submit` input.